pkg/usernames/usernames.go

```diff
@@ -0,0 +1,19 @@
+package usernames
```

Comment:
I know we have a mixture of plural and singular package names but my understanding is that in golang it's preferable to use singular package names.

Reply:
I named it using the plural form mainly because the endpoint will be `/usernames/<username>` . But I agree, in Golang package names are in singular form usually . 

I can rename it.

Reply:
Maybe we should name the struct as `Usernames`. Because this is the actual REST resource type name: `/usernames/...`

Reply:
I haven't named the struct `Usernames` because the package is called already `Usernames` or even if we name it `Username` it will look like `username.Usernames` from a client perspective , which is not a good practice in Go.

While `username.Response` looks better IMO. But I'm open to suggestions.

Reply:
renamed the package to `username` (singular form) . Plz see : [8f86a3c](https://github.com/codeready-toolchain/registration-service/pull/355/commits/8f86a3c62e557249046644ab2d19e521e172cb9d)